moped-editor/src/theme/index.js

```diff
@@ -26,7 +26,7 @@ const theme = createTheme(adaptV4Theme({
     },
     text: {
       primary: "#212121",
-      secondary: "#848484",
+      secondary: "#666666",
       header: "#848484",
```

Comment:
This is outside the scope of the issue, but I noticed that we use `text.header` in only three places in the app where we have tabs: the Dashboard view, the main app header, and in the project view. For some reason, the app header respects `text.header` but the other two do not.

This is making me feel like the header style isn't necessary since it isn't working as expected. What do you think about removing this style and those three places that it is used? All three would then default to the new secondary color that you updated.

Reply:
sounds great - thanks!